src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -116,6 +116,7 @@ trait ScalaSettings extends StandardScalaSettings with Warnings { _: MutableSett
   val elidebelow         = IntSetting          ("-Xelide-below", "Calls to @elidable methods are omitted if method priority is lower than argument",
                                                 elidable.MINIMUM, None, elidable.byName get _)
   val noForwarders       = BooleanSetting      ("-Xno-forwarders", "Do not generate static forwarders in mirror classes.")
+  val useNolink          = BooleanSetting      ("-Xuse-nolink", "Allow generated classes to refer to methods annotated @nolink")
```

Comment:
shouldn't this be a `ScalaVersionSetting` or similar

Reply:
I was wondering what would be a use case for specifying the version in which a method was added, and allowing to filter in the `-Xuse-nolink` flag. If there is one we can add it.